duckduckgo/iOS
# Update embedded Tracker Radar data

Task/Issue URL: https://app.asana.com/0/414709148257752/1200045203766025/f
Tech Design URL:
CC:

**Description**:

This PR updates the bundled Tracker Radar data file.

**Steps to test this PR**:
1. Verify that unit tests pass
1. Verify that integration tests related to content blocking pass
1. Download the latest [tds.json](https://staticcdn.duckduckgo.com/trackerblocking/v2.1/tds.json) file manually and check that the ETag matches the one specified in this PR (it might have changed by then though)
1. Run a fresh install of the app and verify that the tracker data is used correctly



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
